components/alert/alert.css

```diff
@@ -0,0 +1,36 @@
+.alert-area{
+   align-items: center;
+  
+   margin-left: 206px;
+
```

Comment:
u can remove extra spacing by using extension like "remove empty lines" in vs code

Reply:
Thank you for giving the tip which I don't know earlier.